nuxt.config.ts

```diff
@@ -1,4 +1,13 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
+import { resolve } from "path";
 export default defineNuxtConfig({
-  devtools: { enabled: true }
-})
+  $production: {
+    routeRules: {
+      '/**': { isr: true }
+    }
+  },
```

Comment:
don't think we need this - we don't yet have any pages that we need to statically generate